cekit/builders/docker_builder.py

```diff
@@ -195,7 +195,7 @@ def _docker_client(self):
                 )
             )
 
-        params = {"version": "1.22"}
+        params = {}
```

Comment:
Wouldn't this cause problems when the Docker API of newer version would break our usage of it?

Reply:
Even older versions of the library (e.g. 4.0.2 ; https://github.com/docker/docker-py/blob/4.0.2-release/docker/api/client.py#L84 )  ( we're currently on 5.0.3 ) have 1.35 as the default. The problem occurred because we were setting it to a *really* old version.

Reply:
In fact, its been 1.35 for 4 years ( https://github.com/docker/docker-py/commit/abd60aedc7e3df813006919222d86717eb8c6fc2 )

Reply:
Although that appeared to cause test failures ; experimenting with updating the API version instead

Reply:
@goldmann I do wonder if we should document the API version we support?

Reply:
We probably could, but I'm not sure if this is out of interest users since we pretty much hide it when using the Docker builder.